app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,80 @@
+            <path stroke-linecap="round" stroke-linejoin="round" d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5" />
+          </svg>
+        </button>
+      </div>
+      <div class="hidden lg:flex lg:gap-x-12">
+        <a href="/categories" class="text-sm font-semibold leading-6 text-gray-900">Categories</a>
+        <a href="/transactions" class="text-sm font-semibold leading-6 text-gray-900">Transactions</a>
+      </div>
+      <div class="hidden lg:flex lg:flex-1 lg:justify-end">
+      <%if user_signed_in?%>
+        <%= link_to "Log out", destroy_user_session_path, method: :delete, class: "-mx-3 block rounded-lg px-3 py-2.5 text-base font-semibold leading-7 text-gray-900 hover:bg-gray-50" %>        
+      <% else %>
+        <%= link_to "Log in", new_user_session_path, class: "-mx-3 block rounded-lg px-3 py-2.5 text-base font-semibold leading-7 text-gray-900 hover:bg-gray-50" %>
+      <% end%> 
+      </div>
+    </nav>
+    <!-- Mobile menu, show/hide based on menu open state. -->
+    <div class="lg:hidden" role="dialog" aria-modal="true">
+      <!-- Background backdrop, show/hide based on slide-over state. -->
+      <div class="fixed inset-0 z-50"></div>
+      <div class="fixed inset-y-0 right-0 z-50 w-full overflow-y-auto bg-white px-6 py-6 sm:max-w-sm sm:ring-1 sm:ring-gray-900/10">
+        <div class="flex items-center justify-between">
+          <a href="#" class="-m-1.5 p-1.5 flex">
+            <img class="h-8 w-auto" src="https://tailwindui.com/img/logos/mark.svg?color=indigo&shade=600" alt="">
+            <span class="">Tranzact</span>
+          </a>
+          <button type="button" class="-m-2.5 rounded-md p-2.5 text-gray-700">
+            <span class="sr-only">Close menu</span>
+            <svg class="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true">
+              <path stroke-linecap="round" stroke-linejoin="round" d="M6 18L18 6M6 6l12 12" />
+            </svg>
+          </button>
+        </div>
+        <div class="mt-6 flow-root">
+          <div class="-my-6 divide-y divide-gray-500/10">
+            <div class="space-y-2 py-6">
+              <a href="#" class="-mx-3 block rounded-lg px-3 py-2 text-base font-semibold leading-7 text-gray-900 hover:bg-gray-50">Categories</a>
+              <a href="#" class="-mx-3 block rounded-lg px-3 py-2 text-base font-semibold leading-7 text-gray-900 hover:bg-gray-50">Transactions</a>
+            </div>
+            <div class="py-6">
+              <%= link_to "Log out", destroy_user_session_path, class: "-mx-3 block rounded-lg px-3 py-2.5 text-base font-semibold leading-7 text-gray-900 hover:bg-gray-50" %>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </header>
+
+    <main class="prose container mx-auto mt-28 px-5 flex">
+      <%= yield %>
+    </main>
```

Comment:
- [ ] This menu is not working as expected on mobile screens, it is covering the whole screen. Since this is a mobile app, kindly fix the design so that the app works as expected.


![popup](https://github.com/Reggeditt/tranzact/assets/53124548/a6896abb-7907-4ea4-960a-f32a9fe6ada7)